conda/env/specs/__init__.py

```diff
@@ -26,7 +26,7 @@
 @deprecated(
     "25.9",
     "26.3",
-    addendum="Use conda.base.context.plugin_manager.get_environment_specifiers.",
+    addendum="Use conda.base.context.plugin_manager.detect_environment_specifier.",
```

Comment:
It's maybe spicy to be changing this deprecation warning here? Not sure how conda's process for this is.